### ISSUE TITLE
Extracting stem feature map

### ISSUE THREAD

**Issue by ashnair1** [facebookresearch/detectron2#2099]
In the ResNet configs, depending on its type, you can extract feature maps from res2 to res5 or res5 alone. I would like to know what would be the simplest way to get the stem feature map (along with res2 to res5) as well.

It's clear from looking at the forward function [here](https://github.com/facebookresearch/detectron2/blob/0dd2b6579a4ba8d6bdddd3f5d010e9c1b620870d/detectron2/modeling/backbone/resnet.py#L433) that this is supported but I'm unsure how to go about this as setting `MODEL.RESNETS.OUT_FEATURES` to ["stem", "res2", "res3", "res4", "res5"] fails at this [line](https://github.com/facebookresearch/detectron2/blob/0dd2b6579a4ba8d6bdddd3f5d010e9c1b620870d/detectron2/modeling/backbone/resnet.py#L613). 

So what is the recommended way to go about this?

**Comment by ashnair1** [comment]
I was able to do this via adding an if check before the line of failure where the stem field is removed before the res blocks are created and then added back before creating the ResNet. 

Regarding the implementation, I had to create a new backbone just to add the if check. Since there is support for extracting the stem feature map, wouldn't it make more sense to add it in `detectron2.modeling.resnet.py`?
